### DISCUSSION TITLE
Add support for multi-columns in Cloudwatch Logs

### DISCUSSION THREAD

**Discussion by sarahzinger** [grafana/grafana#48776] [Data Sources]
When using AWS Cloudwatch you can get logs organized in multiple columns like so:
![Screen Shot 2022-05-05 at 9 38 27 AM](https://user-images.githubusercontent.com/6620164/166961618-fee8d658-d7bc-4285-b221-8bc6e7dcf607.png)

In the above example you see we select for 3 fields and we get back the logs formatted in 3 columns. In my memories this is a similar pattern to other Logs visualization tools like Kibana. 

But with Grafana, I believe the Logs visualization only supports 2 columns, timestamp, and the log message. 

![Screen Shot 2022-05-05 at 11 52 39 AM](https://user-images.githubusercontent.com/6620164/166963096-1e97f510-c836-4a32-af8b-b4f3f4a1be52.png)

The only way to get this data surfaced at a column level is to use a Table, which is less than ideal because a message is often in JSON form, and it's hard to read JSON in a table format. 
![Screen Shot 2022-05-05 at 11 51 25 AM](https://user-images.githubusercontent.com/6620164/166963133-7d1856e2-c2ca-42ef-a187-1db6e9f346b4.png)

I'm not sure if the answer here is to improve the Logs visualization to allow for multiple column support, or if the answer is to make a new visualization altogether like LogTable or something like that?

**Comment by gabor** [comment]
related discussions:
- https://github.com/grafana/grafana/discussions/42315
- https://github.com/grafana/grafana/discussions/42319

**Comment by sarahzinger** [comment]
closing in favor of https://github.com/grafana/grafana/issues/77949
